code/game/objects/items/weapons/shields.dm

```diff
@@ -48,6 +48,24 @@
 	burn_state = FLAMMABLE
 	block_chance = 30
 
+/obj/item/weapon/shield/riot/makeshift
+	name = "makeshift shield"
+	desc = "A tray with cable tightly wrapped around it to act as a handle."
+	icon_state = "makeshift"
+	item_state = "makeshift"
+	force = 5
+	block_chance = 30
+	materials = list()
+
+/obj/item/weapon/shield/riot/makeshift/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)
+	if(prob(30))
+		visible_message("<span class='danger'>[owner]'s shield breaks apart!</span>", "<span class='userdanger'>Your shield breaks apart!</span>")
+		playsound(usr, 'sound/effects/bang.ogg', 30, 1)
+		usr.unEquip(src)
+		qdel(src)
+	return 0

```

Comment:
Now it will always fail to block.

You need something like

if(final_block_chance)
	if(damage && prob(30))
		shield breaks
	return 1
return 0